c++/src/kj/thread.c++

```diff
@@ -125,7 +125,7 @@ void* Thread::runThread(void* ptr) {
 
 void Thread::ThreadState::unref() {
 #if _MSC_VER
-  if (_InterlockedDecrement_rel(&refcount)) {
+  if (_InterlockedDecrement(&refcount)) {
```

Comment:
The compiler will ensure that `refcount` is 32-bit aligned. (In general, the compiler ensures that all variables are aligned to min(variable size, platform bitness).)

I didn't notice that `_rel` is ARM-only -- that's really weird. I would think they'd make the same API available on x86 and have it alias to the full-barrier version. Oh well.

If you have to remove the `_rel` anyway, then I suppose you can remove the `_ReadBarrier()` on the next line, since it's redundant now.